tensorflow/contrib/seq2seq/BUILD

```diff
@@ -16,6 +16,18 @@ py_library(
 )
 
 cuda_py_test(
+    name = "loss_test",
+    size = "medium",
+    srcs = ["python/kernel_tests/decoder_fn_test.py"],
+    additional_deps = [
+        ":seq2seq_py",
+        "//tensorflow:tensorflow_py",
+        "//tensorflow/python:framework_test_lib",
+        "//tensorflow/python:platform_test",
+    ],
+)
+
+cuda_py_test(
```

Comment:
can you delete layers_test and this build rule?  we don't seem to have a layers file.